c7n/output.py

```diff
@@ -349,6 +382,9 @@ def join_log(self):
         self.handler = self.get_handler()
         self.handler.setLevel(logging.DEBUG)
         self.handler.setFormatter(logging.Formatter(self.log_format))
+        if self.ctx.options.get("log_type") == "json":
+            json_logging.ENABLE_JSON_LOGGING = True
+            json_logging.init(custom_formatter=JSONFormatter)
```

Comment:
what exactly is this doing? ie. if this just sets logging.setFormatter(JSONFormatter) what is missing?